db/db_compaction_test.cc

```diff
@@ -601,7 +601,8 @@ TEST_F(DBCompactionTest, ZeroSeqIdCompaction) {
   CompactionOptions compact_opt;
   compact_opt.compression = kNoCompression;
   compact_opt.output_file_size_limit = 4096;
-  const int key_len = compact_opt.output_file_size_limit / 5;
+  const size_t key_len =
+    static_cast<size_t>(compact_opt.output_file_size_limit) / 5;
```

Comment:
But you see in this test case, output_file_size_limit is 4096. We can safely cast it to 32bit type.